ros2service/ros2service/verb/find.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.node.strategy import NodeStrategy
+from ros2srv.api import service_type_completer
+from ros2service.api import get_service_names_and_types
+from ros2service.verb import VerbExtension
+
+
+class FindVerb(VerbExtension):
+    """Output a list of available services of a given type."""
+
+    def add_arguments(self, parser, cli_name):
+        arg = parser.add_argument(
+            'service_type',
+            help="Name of the ROS service type to filter for "
```

Comment:
This needs to use single quotes to pass the linter.

Reply:
Really ? This passes the colcon test call. Only the first line needs to use single quotes ??

Reply:
> Really ?

Yes, see our developer guide: https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#python

> This passes the colcon test call.

It fails for me and I assume it will in CI. Do you have `flake8-quotes` installed?

> Only the first line needs to use single quotes ??

Yes, because the string literal in the next line contains single quotes and using double quotes avoids the need for escaping them.

Reply:
Done. `test_flake8.py` passes, not sure about `flake8-quotes` I'll check.